src/fs/feature/xattr.rs

```diff
@@ -163,29 +225,42 @@ mod lister {
             }
         }
 
-        pub fn listxattr_second(&self, c_path: &CString, buf: &mut Vec<u8>, bufsize: ssize_t) -> ssize_t {
+        pub fn listxattr_second(&self, c_path: &CString, buf: &mut Vec<u8>, bufsize: size_t) -> ssize_t {
             unsafe {
                 listxattr(
                     c_path.as_ptr(),
-                    buf.as_mut_ptr() as *mut c_char,
-                    bufsize as size_t,
+                    buf.as_mut_ptr().cast::<i8>(),
```

Comment:
Is the `::<i8>` part necessary? Because it will prevent compilation on some ARM platforms, see #888. If removing the turbofish doesn’t, then don’t worry, we’ll deal with it in the other PR anyway.

Reply:
Dropped the type specification